va/va.h

```diff
@@ -2350,6 +2351,8 @@ typedef struct _VAPictureParameterBufferH264
         unsigned int value;
     } pic_fields;
     unsigned short frame_num;
+    unsigned char num_ref_idx_l0_default_active_minus1;
+    unsigned char num_ref_idx_l1_default_active_minus1;
```

Comment:
The point I am trying to make is that I don't think there is any reason to have these values because they are purely syntactic (optimising bit use in the slice header).

Do you disagree with any of the following?

(a)  The standard offers no constraint beyond a 0-31 range - since they can always be overidden in the slice header, they can take any value at all in that regardless of any other property of the stream.  In particular, they don't tell you anything about the number of references - they need not even be <16 when frame_mbs_only_flag is set.  Hence they are of no value directly.

(b)  They don't do anything if you have a VASliceParameterBufferH264 structure.  This supplies the values num_ref_idx_lN_active_minus1 without num_ref_idx_active_override_flag, so the default values for bitstream parsing are irrelevant.

(c)  Parsing the slice header can't make use of them by determining num_ref_idx_active_override_flag, because you don't know enough about the syntax elements present.  Specifically, bottom_field_pic_order_in_frame_present_flag is unknown, so delta_pic_order_cnt_bottom / delta_pic_order_cnt[1] may or may not be present and you can't parse anything after that point.